pyproj/geod.py

```diff
@@ -310,6 +311,61 @@ def fwd(  # pylint: disable=invalid-name
         outz = _convertback(z_data_type, inz)
         return outx, outy, outz
 
+    def fwd_point(  # pylint: disable=invalid-name
+        self,
+        lon: float,
+        lat: float,
+        az: float,
+        dist: float,
+        radians: bool = False,
+    ) -> Tuple[float, float, float]:
+        """
+        Forward transformation
+
+        Determine longitude, latitude and back azimuth of terminus
+        point given longitude and latitude of initial point,
+        plus forward azimuth and distance.
+
+        Accepted numeric scalar:
+
+        - :class:`int`
+        - :class:`float`
+        - :class:`numpy.floating`
+        - :class:`numpy.integer`
+
+        Parameters
+        ----------
+        lon: scalar or array
```

Comment:
I think this is something to look out for in the new docstrings.
```suggestion
        lon: scalar
```